src/js/correlation/models/BrimEvent.ts

```diff
@@ -0,0 +1,21 @@
+import {zng} from "../../../../zealot/dist"
+import {UnknownEvent} from "./UnknownEvent"
+import {SuricataEvent} from "./SuricataEvent"
+import {ZeekEvent} from "./ZeekEvent"
+
+export interface BrimEventInterface {
+  getType: () => string
+  getTime: () => Date
+}
```

Comment:
This is my first step towards some classes that "know" about Zeek and Suricata fields.

Reply:
I don't know if these classes should contain the code that's in the Correlation class. More will be revealed.